documentation.md

```diff
@@ -733,3 +733,34 @@ boxoffice = imdb.BoxOffice()
 | Methods         | Details                                                                       |
 | --------------- | ----------------------------------------------------------------------------- |
 | `.top_movies()` | Returns the top box office movies, weekend and total gross and weeks released |
+
+### Wuzzuf
+
+The `JobScraper` class provides methods for configuring scraping parameters and fetching job listings:
+
+| Methods               | Details                                                                                             |
+| --------------------- | --------------------------------------------------------------------------------------------------- |
+| `.filter_job()`        | Apply filters such as job title, country, city, and range of years of experience.                   |
+| `.fetch_jobs()`        | Fetch job listings from the website based on the applied filters, across multiple pages.            |
+
+```python
+from scrap-up import wuzzuf
+```
+### How to use :
```

Comment:
Remove this how to use section. You can see how other modules are documented. 
You can implement the same way.